sdk/storage/CMakeLists.txt

```diff
@@ -138,9 +138,25 @@ set (AZURE_STORAGE_DATALAKE_SOURCE
 add_library(azure-storage-file-datalake ${AZURE_STORAGE_DATALAKE_HEADER} ${AZURE_STORAGE_DATALAKE_SOURCE})
 target_link_libraries(azure-storage-file-datalake azure-storage-blob)
 
+set (AZURE_STORAGE_SHARES_HEADER
+    inc/shares/shares.hpp
```

Comment:
What is the directory hierarchy in other SDK?

Reply:
5 separate services has it's own inc/src folders. I believe we can change them when we pin down the final decision on folder hierachy.

Reply:
https://github.com/Azure/azure-sdk-for-cpp/issues/339 this issue was opened to track the change.